docs/DeveloperGuide.md

```diff
@@ -167,28 +176,63 @@ Classes used by multiple components are in the `ay2021s1_cs2103_w16_3.finesse.co
 
 This section describes some noteworthy details on how certain features are implemented.
 
-### Find transactions
+### Transactions
+
+#### Find transactions feature
+##### Overview
```

Comment:
![image](https://user-images.githubusercontent.com/5585517/98494440-f7df6f00-2277-11eb-83c9-3901377b8f63.png)

Are the consecutive headings intended?

Reply:
The headings are intended as the Transactions section is split into subsections by feature, while for other sections (e.g. Budgeting), the whole section is one huge feature and need not be split into subsections.